src/libopensc/card-piv.c

```diff
@@ -3636,6 +5717,14 @@ piv_pin_cmd(sc_card_t *card, struct sc_pin_cmd_data *data, int *tries_left)
 	r = iso_drv->ops->pin_cmd(card, data, tries_left);
 	priv->pin_cmd_verify = 0;
 
+	/* tell user veriy not supported on contactless without VCI */
```

Comment:
s/veriy/verify/

Reply:
Adding fix.

Reply:
Fixed in 379a5eeb49adfdb63f6bdd621ca67943f782465a